package.json

```diff
@@ -76,6 +79,9 @@
 		"slick-carousel": "1.8.1",
 		"styled-components": "5.1.1",
 		"styled-media-query": "2.1.2",
+		"synthetix": "2.74.1",
```

Comment:
@koredefashokun wasn't this just removed with the decentralized deployment PR?

Reply:
Thanks for pointing it out. I will remove the package.